README.md

```diff
@@ -257,6 +259,16 @@ Notes:
 
 ### Special classes within teacher content
 
+####Teacher Content with NGSS tag
+
+```html
+
+<note class="os-teacher">
+  <label>Teacher Edition</label>
+  <p>NGSS HS-PS2-1:<span class="ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze
```

Comment:
Phil makes a very good point. We are only targeting the text of the TEKS/NGSS standards, but we should target both the identifier and the text. I will add another commit. 

Phil suggests: <p class=“tag-def”><span class=“tag-name”>NGSS HS-PS2-1</span><span class=“tag-split”>:</span><span class=“tag-desc ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze…</span></p>